scripts/hardfork/create_runtime_config.sh

```diff
@@ -4,16 +4,28 @@ set -e
 
 FORK_CONFIG_JSON=${FORK_CONFIG_JSON:=fork_config.json}
 LEDGER_HASHES_JSON=${LEDGER_HASHES_JSON:=ledger_hashes.json}
+FORKING_FROM_CONFIG_JSON=${FORKING_FROM_CONFIG_JSON:=genesis_ledgers/mainnet.json}
 
 # If not given, the genesis timestamp is set to 10 mins into the future
 GENESIS_TIMESTAMP=${GENESIS_TIMESTAMP:=$(date -u +"%Y-%m-%dT%H:%M:%SZ" -d "10 mins")}
 
+# Pull the original genesis timestamp from the pre-fork config file
+ORIGINAL_GENESIS_TIMESTAMP=$(jq '.genesis.genesis_state_timestamp' $FORKING_FROM_CONFIG_JSON | sed 's/"//g')
+
+DIFFERENCE_IN_SECONDS=$(($(date -d "$GENESIS_TIMESTAMP" "+%s") - $(date -d "$ORIGINAL_GENESIS_TIMESTAMP" "+%s")))
+# TODO: Don't hard-code 180s per slot here!
+DIFFERENCE_IN_SLOTS=$(($DIFFERENCE_IN_SECONDS / 180))
```

Comment:
I suggest to extract `180` to an environment variable so that this script is still usable in testing with non-mainnet parameters